nextcord/ext/commands/core.py

```diff
@@ -423,6 +423,73 @@ def __init__(self, func: Union[
 
         self.checks.extend(self.parent.checks)  # type: ignore
 
+    # Issue #511
+    @property
+    def required_permissions(self) -> Dict[str, bool]:
+        """
+        Returns the permissions required to run this command.
+
+        Returns
+        -------
+        dict
```

Comment:
```suggestion
        Dict[:class:`str`, :class:`bool`]
```
for specificity in docs